### DISCUSSION TITLE
DHCP option 67

### DISCUSSION THREAD

**Discussion by JlnWntr** [ltsp/ltsp#881] [Q&A]
My admin asks which boot-filename he should enter for _option 67_ on the DHCP-server. Why isn't this documented on ltsp.org?
In my` /srv/tfsp` folder are these files:
`ltsp.img ltsp.ipxe memtest.0 snponly.efi undionly.kpxe`

So I guess my admin has to enter `ltsp/ltsp.ipxe`. _Is this correct?_

**Comment by alkisg** [comment]
It's not a single filename. Read the "if" in either dnsmasq.conf or in this page:
https://ltsp.org/guides/isc-dhcp-server/

I.e. depending on the client mode/state, the DHCP server needs to send undionly.kpxe (BIOS), snponly.efi (UEFI), or ltsp.ipxe (the menu, after iPXE has already been loaded).

**Comment by JlnWntr** [comment]
Thanks!
